### ISSUE TITLE
Modified/fixed

### ISSUE THREAD

**Issue by lvp11** [EbenSantuy/esx_bens_drugs#1]
Hey!

What's the difference between this and the original esx_illegal_drugs?

Thanks.

**Comment by LifeGoal** [comment]
Yeah and why not just make a pr to the esx_illegal_drugs repo instead of taking credit of it with your own name?
